lib/consoletest.pm

```diff
@@ -51,13 +50,6 @@ sub post_fail_hook {
     return if get_var('NOLOGS');
     record_avc_selinux_alerts();
     $self->SUPER::post_fail_hook;
-    # at this point the instance is shutdown
-    return if (is_public_cloud() || is_openstack());
-    select_console('log-console');
-    remount_tmp_if_ro;
```

Comment:
Where did you move this code? Or do you just remove it without replacement?

Reply:
This is already executed in `Utils::Logging::export_logs()` which is called from `opensusebasetest::post_fail_hook()` and so it makes no sense to be duplicated also here.

Reply:
Ok, cool